apps/launch.html.markerb

```diff
@@ -7,73 +7,86 @@ order: 10
 redirect_from: /launch/
 ---
 
-With Fly Launch, you can create a brand new app on Fly.io by running this command from the source directory of your project:
+Fly Launch helps you configure and orchestrate your app. 
+
+To create a brand new app on Fly.io, run this command from the source directory of your project:
 
 ```cmd
 fly launch
 ``` 
 
-If you're in a hurry to try your first launch, we also have a condensed [Hands-on](/docs/hands-on/) that launches a super-simple demo app. You might also be interested in our [language- and framework-specific launch guides](/docs/languages-and-frameworks/).
+If you're in a hurry to try your first launch, we have a condensed [Hands-on](/docs/hands-on/) that launches a super-simple demo app. Or check out our [language- and framework-specific launch guides](/docs/languages-and-frameworks/).
 
 ## Ingredients for a successful `fly launch`
 
 Here are the components of a successful launch, ready for the first deployment:
 
-* a way to get a Docker image that Fly.io can use to boot up a Machine with your app running
-* Fly.io-specific [configuration](/docs/reference/configuration/) for the app, in the form of a `fly.toml` file
-* optional platform resources such as [public IP addresses](/docs/reference/services/), [Fly Volumes](/docs/reference/volumes), [app secrets](/docs/reference/secrets/), [Fly Postgres](/docs/postgres/) clusters, integrated resources like [Upstash for Redis](/docs/reference/redis/)
+* A Docker image that Fly Launch can use to boot up a Machine for your app to run on.
+* A `fly.toml` file for your app's Fly.io-specific [configuration](/docs/reference/configuration/).
```

Comment:
I liked putting the "what it is" before "which file it's in," but if you like this better, that's 👍